### PR TITLE
Update Google Oauth guide

### PR DESCRIPTION
**Pull request by ashleywang1**
# Description

Update Google Oauth guide.

# Checklist:

- [x] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [x] If I updated APIs (our protos) or helm values, I ran `make install-go-tools generated-code` to ensure there will be no code diff
- [x] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [x] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by ashleywang1** [comment]
/skip-changelog

Highlighted section: /skip-changelog
Screen Shot 2021-04-05 at 11 31 04 AM

File: docs/content/guides/security/auth/extauth/oauth/google/_index.md [hunk 1]
```diff
@@ -114,7 +115,7 @@ glooctl create secret oauth --namespace gloo-system --name google --client-secre
 
 Now let's create the `AuthConfig` resource that we will use to secure our Virtual Service.
 
-{{< highlight shell "hl_lines=9-16" >}}
+```shell
```
Comment (kdorosh) [inline docs/content/guides/security/auth/extauth/oauth/google/_index.md:118]: why remove the highlighting?

Comment (ashleywang1) [reply docs/content/guides/security/auth/extauth/oauth/google/_index.md:118]: When I looked at it, the section we were highlighting made no sense. We're applying the entire AuthConfig here.

**Review by kdorosh — APPROVED** [review]

**Pull request merged**